e2e-tests/package.json

```diff
@@ -23,7 +22,7 @@
     "eslint-config-prettier": "^9.0.0",
     "eslint-plugin-mocha": "^10.2.0",
     "ethers": "^5.7.2",
```

Comment:
If we are using `1.3.0` we should update to ethers v6. v5 uses < `1.0.0`.

Reply:
and update all the tests to v6